showcase/src/assets/translations/en.json

```diff
@@ -228,6 +228,8 @@
         "CUSTOMIZATION": {
           "DETAILS": "Stark allows you to customize simultaneously the typography settings of Angular Material and Stark easily by adding a simple SCSS map declaration. <br>First, declare the SCSS map and then you need to import the Stark's theme. On this example, a full version of the map is shown but you can also use it partially.",
           "MAP_EXPLAINED": "Here are the corresponding classes and tags:",
+          "CUSTOM_FONT_EXPLAINED": "To set a web font (eg: Roboto) you must first add the font (ttf, woff, ...) to your projects assets and then add it as a font-face to SCSS.",
+          "CDN_FONT_EXPLAINED": "To set a web font from an external source (CDN) an additional step is required, in <code>config/webpack-custom-config.dev.json</code> file add the host/port to the <code>cspFontSrc</code> property For more information check out <a href=\"https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/font-src\" target=\"_blank\" rel=\"noopener\"> CRP: font-src </a>.",
```

Comment:
For this part: "in <code>config/webpack-custom-config.dev.json</code> file add the host/port to the <code>cspFontSrc</code> property", I think it's better if you put a code block like in the other examples of this page so that it's clear what they need to change in the json file and what in the SCSS file